tests/unit/test_compound_ids.py

```diff
@@ -311,3 +311,12 @@ def testExperimentParse(self):
         self.assertEqual(cid.readGroup, "c")
         self.assertEqual(cid.experiment, "d")
         self.verifyParseFailure(idStr, datamodel.ExperimentCompoundId)
+
+    def testVariantSetMetadataCompoundIdParse(self):
```

Comment:
there should be two tests functions for each new compound id... see the analogues in the file